python/src/iceberg/expressions/base.py

```diff
@@ -328,3 +329,46 @@ def eval(self, struct: StructProtocol) -> Any:
             Any: The value at the referenced field's position in `struct`
         """
         return self._accessor.get(struct)
+
+
+class UnboundReference:
+    """A reference not yet bounded to a field in a schema
```

Comment:
I think you want "bound" instead of "bounded".

Reply:
Fixed! Thanks